examples/eight_schools/eight_schools.py

```diff
@@ -0,0 +1,94 @@
+"""implement the stan 8 schools example
+using the recommended non-centred parameterisation
+
+https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started
+http://mc-stan.org/users/documentation/case-studies/divergences_and_bias.html
+we slightly modify the stan example to:
+                                         avoid improper priors
+                                         avoid half-cauchy priors
+make the statement of the model directly comparable
+
+This model has a hierachy and an inferred variance - yet the example is
+very simple - only the Normal distribution is used"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import tensorflow as tf
+from edward.models import Normal, Empirical
+import numpy as np
+
+def main(_):
+    # data
+    J = 8
+    data_y = np.array([28, 8, -3, 7, -1, 1, 18, 12])
+    data_sigma = np.array([15, 10, 16, 11, 9, 11, 10, 18])
+    # end data
+
+    # model definition
+    mu = Normal(tf.zeros([1]), tf.fill([1], 10.))
```

Comment:
If `mu` is scalar, you can also just use
```python
mu = Normal(0., 10.)
```
Same goes for `logtau`.

Reply:
I seem to need to use: 
mu = Normal([0.], [1.])